ruby/github_account.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'error'
+require_relative 'response_handler'
+class GithubAccount
+  include ResponseHandler
+
+  attr_reader :account_name
+  def initialize(account_name)
+    @account_name = account_name
+  end
+
+  def url
+    "https://api.github.com/users/#{account_name}"
+  end
+
+  def exists?
+    response = parse_response(url)
```

Comment:
`parse_response(url)`
Can we think of a better name for this function?

Is the function strictly parsing a response?
Or is it making a request and then parsing a response?